src/models/routineDao.js

```diff
@@ -0,0 +1,34 @@
+const { AppDataSource } = require("./dataSource");
+
+const routineStartDao = async (id) => {
```

Comment:
```suggestion
const getExerciseRoutineByRoutinId = async (id) => {
```
- Dao에서 선언되는 함수는 특히나 'Create, Read(Get), update, delete' 로 시작하는 함수 외에는 거의 없다고 보시면 됩니다. Entity를 곧바로 관리하는 함수이기 때문입니다.
- 어떤 Entity(Table 이름) 를 어떻게 (CRUD 중 어떤 동작으로 조작하는지) 가공하는지를 함수명에서 드러내시는 것이 좋습니다 :)